### ISSUE TITLE
Anemia of prematurity

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6160]
**Preferred term label:**
Anemia of prematurity

**Synonyms**
Anaemia of prematurity, Premature anemia, Anemia in premature infants, Anemia in preterm infants, AOP

**Definition (free text, please give PubMed ID)**
A decline in haemoglobin concentration in ELBW newborns. It is usually associated with abnormal clinical signs and requires allogeneic RBC transfusions. AOP is characterized by reduced endogenous erythropoietin (EPO), reduced RBC lifespan and hypo-regenerative bone marrow.
PMID: 31588563; DOI: https://doi.org/10.1016/j.ijpam.2015.10.001 

**Parent term (use hpo.jax.org/app)**
Anemia

**Comment by pnrobinson** [comment]
Our rule is that the HPO covers phenotypic features but not etiology. The combination of phenotype and etiology is conceptualized as a disease (https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0001239).
